node/src/Transaction.ts

```diff
@@ -3208,6 +3211,34 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
         return this.addAndReturn(createFunctionStats());
     }
 
+    /**
+     * Returns the serialized payload of all loaded libraries.
+     *
+     * @see {@link https://valkey.io/commands/function-dump/|valkey.io} for details.
+     * @remarks Since Valkey version 7.0.0.
+     * @remarks To execute a transaction with a `functionDump` command, the `exec` command requires `Decoder.Bytes` to handle the response.
```

Comment:
I thought we were going to go with the same solution as Java?

Reply:
Yes we have a task for this